docs/LogGuide.md

```diff
@@ -838,47 +848,71 @@ Network ability lines are a combination of raw network data
 and frequently sampled data from memory
 (e.g. the `66480` current hp value and `-653.0394` x position).
 
-This means that there's a number of caveats going on to handling all the data in these lines.  The raw network data is subject to change over time from ff14 servers.  Also, the data from memory may be slightly stale and out of date
+This means that there's a number of caveats going on to handling all the data in these lines. The raw network data is
+subject to change over time from ff14 servers. Also, the data from memory may be slightly stale and out of date
+
+#### Action Effects
+
+Each ability may have one or more effects. These are indicated by the flagsX and damageX fields, between `targetName`
+and `targetCurHp`. There are eight pairs, each with a 'flags' field and a 'damage' field. The damage field is not
+necessarily always damage. For example, for a buff application, it indicates which buff ID is about to be applied.
+
+This means that damage is not necessarily the first flag. If you are simply trying to determine how much damage a
```

Comment:
Do you have examples of this not being true? I feel like I've seen the damage being first almost always.

Reply:
A few of the existing examples show that already. This is just a generalized version of the "special case shifts" section that was there before.

Reply:
Fair, but I haven't seen the "special case shifts" in a while and was thinking about removing it entirely.  Most of those examples are ancient at this point.